crates/solidity/testing/perf/npm/src/solidity.parser.runner.mts

```diff
@@ -0,0 +1,43 @@
+import { parse } from "@solidity-parser/parser";
+import { log, readRepoFile, resolveImport, Runner, Timing } from "./common.mjs";
+
+export class SolidityParserRunner implements Runner {
```

Comment:
WDYT of using "antlr" instead of "solidity-parser" for this runner?
The "antlr" codename is more unique, and less likely to be confused with Slang's or Solc's own parsers.